scripts/launch/staging.sh

```diff
@@ -1,10 +1,14 @@
 #!/usr/bin/env bash
 
 stack exec -- cardano-node \
-    --system-start 1499246772 \
+    --system-start 1504019546 \
     --log-config log-config-prod.yaml \
     --logs-prefix "logs/qanet" \
+    --keyfile secret.key \
     --db-path db-qanet \
     --wallet-db-path wdb-qanet \
-    --no-ntp \
```

Comment:
Can you justify why you decided to enable NTP? I recall it was quite unreliable.

Reply:
Is no-ntp the default now? 🤔